src/utils/StorageUtils.ts

```diff
@@ -30,7 +30,7 @@ export const GLOBAL_ASSOC_KEY = "";
 
 export default class StorageUtils {
   public static getStorageKey(type: Assoc, id?: string): string {
-    const hashedID = id ? HashUtils.hashString(id) : "";
+    const hashedID = id ? HashUtils.hashString(id) : GLOBAL_ASSOC_KEY;
```

Comment:
Are there any scenarios in which this default actually gets used? Is there any risk of this key change causing any previously-written records to be missed?

Reply:
The default empty string is the same as the `GLOBAL_ASSOC_KEY` which has always only been used for global assocs (e.g. registered keys `SDK_KEYS`). This change is just simplifying that by sharing the same constant.